src/Property/Charset.php

```diff
@@ -103,6 +103,8 @@ public function atRuleName()
 
     /**
      * @return string
+     *
+     * @deprecated since V8.8.0, will be removed in version 9.0.0. Use the specific getter instead.
```

Comment:
This doesn't have a getter for the `CSSString` object property, only one that returns a native `string`.  I wonder if we should first add a getter for the `CSSString`.